composer.json

```diff
@@ -10,7 +10,8 @@
     "license": ["LGPL-3.0-or-later"],
     "require": {
         "php": "^8.2",
-        "ext-mbstring": "*"
+        "ext-mbstring": "*",
+	"psr/container": "^2.0"
```

Comment:
Not sure about this dependency. I don't think it's necessary for the code to work, is it?

Reply:
No it's not necessary for the instanceof check. However, phpstan will fail if the interface is not present. Therefore I would suggest to add it as dev dependency.

**Side note:** `psr/container` is already present in fluid because it's a dependency of `symfony/service-contracts`, which is required by `symfony/console`, which is required by `friendsofphp/php-cs-fixer`.